src/state/state.ts

```diff
@@ -19,16 +11,16 @@ interface GasPriceInfo {
 
 export interface State {
   config: Config;
-  gasPrices: Record<ChainId, Record<ProviderName, GasPriceInfo[]>>;
+  gasPrices: Record<string /* Chain ID */, Record<string /* Provider name */, GasPriceInfo[]>>;
   // The timestamp of when we last detected that the feed requires an update. Note, that if the feed requires an update
   // consecutively, the timestamp is not updated until the feed stops being updatable again.
   firstMarkedUpdatableTimestamps: Record<
-    ChainId,
-    Record<ProviderName, Record<Address /* Sponsor wallet */, number | null>>
+    string /* Chain ID */,
+    Record<string /* Provider name */, Record<Address /* Sponsor wallet */, number | null>>
   >;
-  derivedSponsorWallets: Record<DapiNameOrDataFeedId, PrivateKey>;
-  signedDatas: Record<BeaconId, SignedData>;
-  signedApiUrls: Record<ChainId, Record<ProviderName, SignedApiUrl[]>>;
+  derivedSponsorWallets: Record<string /* dAPI name or data feed ID */, Hex /* Private key */>;
+  signedDatas: Record<Hex /* Beacon ID */, SignedData>;
```

Comment:
[Keccak256Hash](https://github.com/api3dao/commons/blob/fb07e2c083cf63bc35d500aca37eb3fb7ee411c8/src/blockchain-utilities/schema.ts#L26C13-L26C26) for beacon Ids?